frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -449,6 +516,22 @@ function applyChartLineBarSettings(
         forceCenterBar || settings["graph.x_axis.scale"] !== "ordinal",
       );
   }
+
+  // FIXME scoped the chart-body
+  // FIXME should this be done in PostRender.js?
+  if (chartType === "waterfall") {
+    chart.on("pretransition", function(chart) {
+      chart
+        .selectAll("svg g g.chart-body g.stack._0 rect.bar")
+        .style("fill", "transparent");
+      chart
+        .selectAll("svg g g.chart-body g.stack._1 rect.bar")
+        .style("fill", "red");
+      chart
+        .selectAll("svg g g.chart-body g.stack._2 rect.bar")
+        .style("fill", "green");
+    });
+  }
```

Comment:
While the invisible beam needs to be, well, invisible, the green and red bars likely need to be configurable (the color).